spec/spec_helper.rb

```diff
@@ -151,24 +120,19 @@ def main_app
     c.syntax = :expect
   end
 
-  # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = File.expand_path("../fixtures", __FILE__)
-
   config.use_transactional_fixtures = false
 
   config.before :suite do
+    Hyrax::RedisEventStore.instance.redis.flushdb
     DatabaseCleaner.clean_with(:truncation)
     # Noid minting causes extra LDP requests which slow the test suite.
     Hyrax.config.enable_noids = false
+    # Don't use the nested relationship reindexer. Null is much faster
+    Hyrax.config.nested_relationship_reindexer = ->(id:, extent:) {}
```

Comment:
it feels like this should be part of a app test harness as well as I bet it speeds things up a lot. any thoughts on how to do that in an accessible way?

Reply:
it would be easy enough to move something like this into `lib/hyrax/spec/`... the tricky bit is how we communicate with folks about what's available there.

there's starting to be a lot of good stuff shipping with Hyrax to support application test suites, and i think it's probably the case that few people really know about it. 

dev congress topic?